Modelica/Fluid/Dissipation.mo

```diff
@@ -1136,10 +1136,6 @@ Calculation of the mean convective heat transfer coefficient <b> kc </b> for the
         "Input record for function kc_flatTube and kc_flatTube_KC"
         extends Modelica.Icons.Record;
 
-      protected
-        type TYP =
-            Modelica.Fluid.Dissipation.Utilities.Types.HTXGeometry_flatTubes;
-
       public
```

Comment:
That is correct, but I wanted to just have the minimal diff for the discussion. Otherwise everything in the function would be red due to indentation